### ISSUE TITLE
Super Mario 64 DS EUR hangs when obtaining first star

### ISSUE THREAD

**Issue by RodriTaku** [ahezard/nds-bootstrap#144]
Super Mario 64 DS EUR (0022) hangs the moment you get the first star after defeating King Bob-omb in Bob-omb Battlefield mission. This does not occur with Super Mario 64 DS US (v.1.1) (0056).

I'm using a N3DS with TWLoader v5.5.1, nds-bootstrap v0.6.0 (Rev 02) and several unofficial releases (I just checked with 5654c2b). ARM9 speed is set to default and I'm using Mario Kart DS U (0168) as donor.

Testing is really annoying, as you essentially need to make your way from the castle's garden to the top of the battlefield and defeat King Bob-omb every single time...

**Comment by Robz8** [comment]
Does that issue occur on an emulator for PC?

**Comment by RodriTaku** [comment]
Nope. I just tried the same rom with DeSmuME (git#2379dc1_x64) and it worked fine.

**Comment by Robz8** [comment]
If your default ARM9 speed is TWL, see if NTR speed fixes the issue.

**Comment by RodriTaku** [comment]
Nope, my default speed is NTR. I've also tried forcing TWL with the same results. Can anyone else reproduce the issue?

**Comment by Robz8** [comment]
^ Can anyone test the ROM on a flashcard as well?

**Comment by maorninja** [comment]
@RodriTaku I don't get this bug. If I had your specific ROM, I would test it, but no piracy here.

(If you do want to give me your rom, just email me: maorninja@outlook.com )

**Comment by RodriTaku** [comment]
Ok, guys, sorry! It was indeed the rom. I had two different versions of the same rom and I thought I had tried both. As it turns out one them was corrupted. After loading the rom I had in the N3DS in DeSmuME and trying for the n-th time to get the star, this time it did get stuck just as described. I loaded the one that worked when I tried the other day in DeSmuME in my SD and it worked. Anyway, mistery solved and sorry for the waste of time.
